http/impl/src/main/java/cloud/piranha/http/impl/DefaultHttpServer.java

```diff
@@ -141,6 +141,21 @@ public DefaultHttpServer(int serverPort, HttpServerProcessor processor, boolean
         this.ssl = ssl;
     }
 
+
+    /**
+     * Constructor
+     *
+     * @param serverPort the server port.
+     * @param processor the HTTP server processor.
+     */
+    public DefaultHttpServer(int serverPort, HttpServerProcessor processor) {
```

Comment:
Please use the existing constructor that takes in a boolean to enable/disable SSL